### PR TITLE
Revert "Use sdx-common transformers"

### PR DESCRIPTION
**Pull request by warrenbailey**
Reverting the move towards sdx-common because the transformers in common are not correct

### CODE REVIEW

**Review by dcdarrell9 — APPROVED** [review]

File: transform/__init__.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 app = Flask(__name__)
 
+from .views import test_views  # noqa
```
Comment (JamesGardiner) [inline transform/__init__.py:5]: Won't this import test_views whenever an app object is created? Might not be the best way to do this for production.

File: transform/transformers/ImageTransformer.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+import subprocess
+import sys
+import zipfile
+
+from requests.packages.urllib3.exceptions import MaxRetryError
+
+try:
+    from PDFTransformer import PDFTransformer
+except ImportError:
+    from .PDFTransformer import PDFTransformer
```
Comment (JamesGardiner) [inline transform/transformers/ImageTransformer.py:23]: Surely we can dispense with the `try` block here, as we know we have a `PDFTransformer` module?

File: transform/views/main.py [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-import pkg_resources
 from structlog import wrap_logger
+from flask import request, make_response, send_file, jsonify
+from transform.transformers import PDFTransformer, ImageTransformer, CORATransformer
+from jinja2 import Environment, PackageLoader
 
-from transform import app
-from transform import settings
-from transform.transformers import CORATransformer
+import json
```
Comment (JamesGardiner) [inline transform/views/main.py:9]: Imports aren't pep8 compliant

File: transform/views/test_views.py [hunk 4]
```diff
@@ -1,16 +1,13 @@
...
+from transform.transformers import PDFTransformer, ImageTransformer
 from transform import app
 from transform import settings
+from jinja2 import Environment, PackageLoader
+
+from flask import make_response, send_file
+import logging
+from structlog import wrap_logger
+import json
+import os.path
```
Comment (JamesGardiner) [inline transform/views/test_views.py:10]: pep8 again

File: transform/views/test_views.py [hunk 5]
```diff
@@ -216,6 +213,23 @@ def images_test():
         return send_file(zipfile, mimetype='application/zip')
 
 
+@app.route('/pdf-test', methods=['GET'])
```
Comment (JamesGardiner) [inline transform/views/test_views.py:216]: Again, I'm not sure having test_routes in a production app is a good idea. Just more scope for errors.

**Review by JamesGardiner — DISMISSED** [review]
I've given this a cursory glance, but haven't been thorough with the parts that are copy/pasted from sdx-common

**Comment by warrenbailey** [comment]
@JamesGardiner This PR is a github revert, which takes us back to exactly the same state it was before the last PR went in.

Your comments are valid but I'd rather do them in a separate PR so we don't pollute the revert.

**Pull request merged**